pages/markets/[marketid].tsx

```diff
@@ -137,17 +131,9 @@ const Market: NextPage<{
       return <NotFoundPage backText="Back To Markets" backLink="/" />;
     }
 
-    //required to fix title element warning
-    const question = indexedMarket.question;
```

Comment:
Did you verify that there is no longer an error after removing this?

Reply:
I moved this directly into the alt tag property and it doesn't seem to show an error in the console.